zkevm-circuits/src/taiko_super_circuit.rs

```diff
@@ -339,16 +342,7 @@ impl<F: Field> Circuit<F> for SuperCircuit<F> {
             self.block
                 .sha3_inputs
                 .iter()
-                .chain(std::iter::once(&self.pi_circuit.public_data.rpi_bytes()))
-                .chain(
-                    &self
-                        .block
-                        .bytecodes
-                        .clone()
-                        .into_iter()
-                        .map(|b| b.1.bytes)
-                        .collect_vec(),
-                ),
+                .chain(std::iter::once(&self.pi_circuit.evidence.encode_raw())),
```

Comment:
I would think the bytecodes still need to be added here.

Reply:
Yes, otherwise some tests fails.